platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r73/DeprecationsCrossVersionSpec.groovy

```diff
@@ -113,8 +111,7 @@ class DeprecationsCrossVersionSpec extends ToolingApiSpecification {
         }
 
         then:
-        def e = thrown(org.gradle.tooling.BuildException)
-        e.message.contains("Could not fetch model of type 'List'")
-        e.cause.message.contains("Resolution of the configuration :a:compileClasspath was attempted from a context different than the project context.")
+        assertHasConfigureSuccessfulLogging(true)
```

Comment:
So these ignores were all added as temporary until a new version of Asciidoctor was ready.  Our dep has since been updated, so I removed the ignores and updated the test expectations to match the current behavior - which is to throw a deprecation.

I didn't look into why they were failing.  I figure this way, when we remove the deprecation in Gradle 9.0, these will break again and we can figure out if the behavior needs is correct or not - this just keeps track of what is happening now.

Reply:
This test runs against Gradle 8.0 through current, does it no longer fail and emit deprecations for all of them?

Reply:
We talked about this and decided we could just remove this test entirely and change the TargetGradleVersion bounds on the test above this one.